packages/SwingSet/src/kernel/json-stable-stringify.js

```diff
@@ -13,10 +13,10 @@ const isArray =
 
 const objectKeys =
   Object.keys ||
-  function(obj) {
+  function (obj) {
```

Comment:
This change I do not like. OTOH, we're moving away from `function` functions anyway, so no worries.